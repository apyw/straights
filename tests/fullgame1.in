h
c
c
c
play 7C
play 8D
play TD
play 8H
play 9H
play TH
play QH
play KH
play 5C
play 3H
play 3C
play 2D
discard 3S
play 8S
play 7D
play QS
play KS
play 8D
play 9D
play JH
play 8C
play TC
play 6H
play 3C
discard 2D
play 3D
play 7H
play 6C
play TH
discard AS
discard AD
play 3H
discard 2D
discard AC
play 9D
play JD
play 5D
play 4D
play QS
play 7C
play 5C
play 9H
play JH
play KH
play 8D
play TD
play JD
play JS
play 5S
play 5D
play 3H
play 3D
play 8S
play TS
play QS
play 7D
play 7H
play 8H
play TH
discard AD
play 3D
play 2H
play 2C
play AH
discard QH
