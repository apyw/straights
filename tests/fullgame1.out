Is Player1 a human (h) or a computer (c)?
>Is Player2 a human (h) or a computer (c)?
>Is Player3 a human (h) or a computer (c)?
>Is Player4 a human (h) or a computer (c)?
>A new round begins. It's Player4's turn to play.
Player4 plays 7S.
Cards on the table:
Clubs:
Diamonds:
Hearts:
Spades: 7
Your hand: 9H 3H KH 7C 3S 8H 5C QH 8D TH 2D TD 3C
Legal plays: 7C
>Player1 plays 7C.
Player2 plays 7D.
Player3 plays 6C.
Player4 plays 6D.
Cards on the table:
Clubs: 6 7
Diamonds: 6 7
Hearts:
Spades: 7
Your hand: 9H 3H KH 3S 8H 5C QH 8D TH 2D TD 3C
Legal plays: 5C 8D
>Player1 plays 8D.
Player2 plays 8C.
Player3 plays 9D.
Player4 plays 9C.
Cards on the table:
Clubs: 6 7 8 9
Diamonds: 6 7 8 9
Hearts:
Spades: 7
Your hand: 9H 3H KH 3S 8H 5C QH TH 2D TD 3C
Legal plays: 5C TD
>Player1 plays TD.
Player2 plays 7H.
Player3 plays JD.
Player4 plays 8S.
Cards on the table:
Clubs: 6 7 8 9
Diamonds: 6 7 8 9 T J
Hearts: 7
Spades: 7 8
Your hand: 9H 3H KH 3S 8H 5C QH TH 2D 3C
Legal plays: 8H 5C
>Player1 plays 8H.
Player2 plays 6H.
Player3 plays 9S.
Player4 plays TS.
Cards on the table:
Clubs: 6 7 8 9
Diamonds: 6 7 8 9 T J
Hearts: 6 7 8
Spades: 7 8 9 T
Your hand: 9H 3H KH 3S 5C QH TH 2D 3C
Legal plays: 9H 5C
>Player1 plays 9H.
Player2 plays 5H.
Player3 plays 4H.
Player4 plays 5D.
Cards on the table:
Clubs: 6 7 8 9
Diamonds: 5 6 7 8 9 T J
Hearts: 4 5 6 7 8 9
Spades: 7 8 9 T
Your hand: 3H KH 3S 5C QH TH 2D 3C
Legal plays: 3H 5C TH
>Player1 plays TH.
Player2 plays JH.
Player3 plays TC.
Player4 plays 4D.
Cards on the table:
Clubs: 6 7 8 9 T
Diamonds: 4 5 6 7 8 9 T J
Hearts: 4 5 6 7 8 9 T J
Spades: 7 8 9 T
Your hand: 3H KH 3S 5C QH 2D 3C
Legal plays: 3H 5C QH
>Player1 plays QH.
Player2 plays JC.
Player3 plays 3D.
Player4 plays QC.
Cards on the table:
Clubs: 6 7 8 9 T J Q
Diamonds: 3 4 5 6 7 8 9 T J
Hearts: 4 5 6 7 8 9 T J Q
Spades: 7 8 9 T
Your hand: 3H KH 3S 5C 2D 3C
Legal plays: 3H KH 5C 2D
>Player1 plays KH.
Player2 plays JS.
Player3 plays QS.
Player4 plays KS.
Cards on the table:
Clubs: 6 7 8 9 T J Q
Diamonds: 3 4 5 6 7 8 9 T J
Hearts: 4 5 6 7 8 9 T J Q K
Spades: 7 8 9 T J Q K
Your hand: 3H 3S 5C 2D 3C
Legal plays: 3H 5C 2D
>Player1 plays 5C.
Player2 plays 4C.
Player3 plays KC.
Player4 plays 6S.
Cards on the table:
Clubs: 4 5 6 7 8 9 T J Q K
Diamonds: 3 4 5 6 7 8 9 T J
Hearts: 4 5 6 7 8 9 T J Q K
Spades: 6 7 8 9 T J Q K
Your hand: 3H 3S 2D 3C
Legal plays: 3H 2D 3C
>Player1 plays 3H.
Player2 plays QD.
Player3 discards 4S.
Player4 plays 2H.
Cards on the table:
Clubs: 4 5 6 7 8 9 T J Q K
Diamonds: 3 4 5 6 7 8 9 T J Q
Hearts: 2 3 4 5 6 7 8 9 T J Q K
Spades: 6 7 8 9 T J Q K
Your hand: 3S 2D 3C
Legal plays: 2D 3C
>Player1 plays 3C.
Player2 plays KD.
Player3 plays AH.
Player4 plays 5S.
Cards on the table:
Clubs: 3 4 5 6 7 8 9 T J Q K
Diamonds: 3 4 5 6 7 8 9 T J Q K
Hearts: A 2 3 4 5 6 7 8 9 T J Q K
Spades: 5 6 7 8 9 T J Q K
Your hand: 3S 2D
Legal plays: 2D
>Player1 plays 2D.
Player2 discards 2S.
Player3 plays 2C.
Player4 plays AD.
Cards on the table:
Clubs: 2 3 4 5 6 7 8 9 T J Q K
Diamonds: A 2 3 4 5 6 7 8 9 T J Q K
Hearts: A 2 3 4 5 6 7 8 9 T J Q K
Spades: 5 6 7 8 9 T J Q K
Your hand: 3S
Legal plays:
>Player1 discards 3S.
Player2 discards AS.
Player3 plays AC.
Player1's discards: 3S
Player1's score: 0 + 3 = 3
Player2's discards: 2S AS
Player2's score: 0 + 3 = 3
Player3's discards: 4S
Player3's score: 0 + 4 = 4
Player4's discards:
Player4's score: 0 + 0 = 0
A new round begins. It's Player3's turn to play.
Player3 plays 7S.
Player4 discards QH.
Cards on the table:
Clubs:
Diamonds:
Hearts:
Spades: 7
Your hand: JH 8S 3D 3C TC 6H 8C 9D 2D QS 8D 7D KS
Legal plays: 8S 7D
>Player1 plays 8S.
Player2 plays 9S.
Player3 discards AH.
Player4 plays TS.
Cards on the table:
Clubs:
Diamonds:
Hearts:
Spades: 7 8 9 T
Your hand: JH 3D 3C TC 6H 8C 9D 2D QS 8D 7D KS
Legal plays: 7D
>Player1 plays 7D.
Player2 plays 7C.
Player3 plays JS.
Player4 discards 3H.
Cards on the table:
Clubs: 7
Diamonds: 7
Hearts:
Spades: 7 8 9 T J
Your hand: JH 3D 3C TC 6H 8C 9D 2D QS 8D KS
Legal plays: 8C QS 8D
>Player1 plays QS.
Player2 plays 6C.
Player3 discards AC.
Player4 discards 4S.
Cards on the table:
Clubs: 6 7
Diamonds: 7
Hearts:
Spades: 7 8 9 T J Q
Your hand: JH 3D 3C TC 6H 8C 9D 2D 8D KS
Legal plays: 8C 8D KS
>Player1 plays KS.
Player2 plays 5C.
Player3 discards JC.
Player4 discards 3S.
Cards on the table:
Clubs: 5 6 7
Diamonds: 7
Hearts:
Spades: 7 8 9 T J Q K
Your hand: JH 3D 3C TC 6H 8C 9D 2D 8D
Legal plays: 8C 8D
>Player1 plays 8D.
Player2 plays 7H.
Player3 plays 8H.
Player4 plays 9H.
Cards on the table:
Clubs: 5 6 7
Diamonds: 7 8
Hearts: 7 8 9
Spades: 7 8 9 T J Q K
Your hand: JH 3D 3C TC 6H 8C 9D 2D
Legal plays: 6H 8C 9D
>Player1 plays 9D.
Player2 plays TD.
Player3 discards 4H.
Player4 plays TH.
Cards on the table:
Clubs: 5 6 7
Diamonds: 7 8 9 T
Hearts: 7 8 9 T
Spades: 7 8 9 T J Q K
Your hand: JH 3D 3C TC 6H 8C 2D
Legal plays: JH 6H 8C
>Player1 plays JH.
Player2 plays 4C.
Player3 discards 2S.
Player4 discards 5H.
Cards on the table:
Clubs: 4 5 6 7
Diamonds: 7 8 9 T
Hearts: 7 8 9 T J
Spades: 7 8 9 T J Q K
Your hand: 3D 3C TC 6H 8C 2D
Legal plays: 3C 6H 8C
>Player1 plays 8C.
Player2 plays JD.
Player3 plays 9C.
Player4 plays QD.
Cards on the table:
Clubs: 4 5 6 7 8 9
Diamonds: 7 8 9 T J Q
Hearts: 7 8 9 T J
Spades: 7 8 9 T J Q K
Your hand: 3D 3C TC 6H 2D
Legal plays: 3C TC 6H
>Player1 plays TC.
Player2 plays 6S.
Player3 plays KD.
Player4 discards 2H.
Cards on the table:
Clubs: 4 5 6 7 8 9 T
Diamonds: 7 8 9 T J Q K
Hearts: 7 8 9 T J
Spades: 6 7 8 9 T J Q K
Your hand: 3D 3C 6H 2D
Legal plays: 3C 6H
>Player1 plays 6H.
Player2 plays 5S.
Player3 discards AD.
Player4 discards 2C.
Cards on the table:
Clubs: 4 5 6 7 8 9 T
Diamonds: 7 8 9 T J Q K
Hearts: 6 7 8 9 T J
Spades: 5 6 7 8 9 T J Q K
Your hand: 3D 3C 2D
Legal plays: 3C
>Player1 plays 3C.
Player2 plays 6D.
Player3 discards KC.
Player4 plays 5D.
Cards on the table:
Clubs: 3 4 5 6 7 8 9 T
Diamonds: 5 6 7 8 9 T J Q K
Hearts: 6 7 8 9 T J
Spades: 5 6 7 8 9 T J Q K
Your hand: 3D 2D
Legal plays:
>Player1 discards 2D.
Player2 plays 4D.
Player3 discards QC.
Player4 discards AS.
Cards on the table:
Clubs: 3 4 5 6 7 8 9 T
Diamonds: 4 5 6 7 8 9 T J Q K
Hearts: 6 7 8 9 T J
Spades: 5 6 7 8 9 T J Q K
Your hand: 3D
Legal plays: 3D
>Player1 plays 3D.
Player2 discards KH.
Player1's discards: 2D
Player1's score: 3 + 2 = 5
Player2's discards: KH
Player2's score: 3 + 13 = 16
Player3's discards: AH AC JC 4H 2S AD KC QC
Player3's score: 4 + 45 = 49
Player4's discards: QH 3H 4S 3S 5H 2H 2C AS
Player4's score: 0 + 32 = 32
A new round begins. It's Player3's turn to play.
Player3 plays 7S.
Player4 plays 7C.
Cards on the table:
Clubs: 7
Diamonds:
Hearts:
Spades: 7
Your hand: 6C 9D 5D QS 2D AS 7H AC JD AD 4D 3H TH
Legal plays: 6C 7H
>Player1 plays 7H.
Player2 plays 8C.
Player3 plays 8H.
Player4 plays 6H.
Cards on the table:
Clubs: 7 8
Diamonds:
Hearts: 6 7 8
Spades: 7
Your hand: 6C 9D 5D QS 2D AS AC JD AD 4D 3H TH
Legal plays: 6C
>Player1 plays 6C.
Player2 plays 9C.
Player3 plays 9H.
Player4 plays 5C.
Cards on the table:
Clubs: 5 6 7 8 9
Diamonds:
Hearts: 6 7 8 9
Spades: 7
Your hand: 9D 5D QS 2D AS AC JD AD 4D 3H TH
Legal plays: TH
>Player1 plays TH.
Player2 plays 5H.
Player3 plays 8S.
Player4 plays JH.
Cards on the table:
Clubs: 5 6 7 8 9
Diamonds:
Hearts: 5 6 7 8 9 T J
Spades: 7 8
Your hand: 9D 5D QS 2D AS AC JD AD 4D 3H
Legal plays:
>Player1 discards AS.
Player2 plays 6S.
Player3 plays 5S.
Player4 plays QH.
Cards on the table:
Clubs: 5 6 7 8 9
Diamonds:
Hearts: 5 6 7 8 9 T J Q
Spades: 5 6 7 8
Your hand: 9D 5D QS 2D AC JD AD 4D 3H
Legal plays:
>Player1 discards AD.
Player2 plays 4S.
Player3 plays 4H.
Player4 plays 3S.
Cards on the table:
Clubs: 5 6 7 8 9
Diamonds:
Hearts: 4 5 6 7 8 9 T J Q
Spades: 3 4 5 6 7 8
Your hand: 9D 5D QS 2D AC JD 4D 3H
Legal plays: 3H
>Player1 plays 3H.
Player2 plays TC.
Player3 plays 2S.
Player4 plays JC.
Cards on the table:
Clubs: 5 6 7 8 9 T J
Diamonds:
Hearts: 3 4 5 6 7 8 9 T J Q
Spades: 2 3 4 5 6 7 8
Your hand: 9D 5D QS 2D AC JD 4D
Legal plays:
>Player1 discards 2D.
Player2 plays QC.
Player3 plays 7D.
Player4 plays KH.
Cards on the table:
Clubs: 5 6 7 8 9 T J Q
Diamonds: 7
Hearts: 3 4 5 6 7 8 9 T J Q K
Spades: 2 3 4 5 6 7 8
Your hand: 9D 5D QS AC JD 4D
Legal plays:
>Player1 discards AC.
Player2 plays 6D.
Player3 plays 8D.
Player4 discards 2C.
Cards on the table:
Clubs: 5 6 7 8 9 T J Q
Diamonds: 6 7 8
Hearts: 3 4 5 6 7 8 9 T J Q K
Spades: 2 3 4 5 6 7 8
Your hand: 9D 5D QS JD 4D
Legal plays: 9D 5D
>Player1 plays 9D.
Player2 plays KC.
Player3 plays 4C.
Player4 plays TD.
Cards on the table:
Clubs: 4 5 6 7 8 9 T J Q K
Diamonds: 6 7 8 9 T
Hearts: 3 4 5 6 7 8 9 T J Q K
Spades: 2 3 4 5 6 7 8
Your hand: 5D QS JD 4D
Legal plays: 5D JD
>Player1 plays JD.
Player2 plays QD.
Player3 plays 9S.
Player4 plays 3C.
Cards on the table:
Clubs: 3 4 5 6 7 8 9 T J Q K
Diamonds: 6 7 8 9 T J Q
Hearts: 3 4 5 6 7 8 9 T J Q K
Spades: 2 3 4 5 6 7 8 9
Your hand: 5D QS 4D
Legal plays: 5D
>Player1 plays 5D.
Player2 plays KD.
Player3 plays 2H.
Player4 plays TS.
Cards on the table:
Clubs: 3 4 5 6 7 8 9 T J Q K
Diamonds: 5 6 7 8 9 T J Q K
Hearts: 2 3 4 5 6 7 8 9 T J Q K
Spades: 2 3 4 5 6 7 8 9 T
Your hand: QS 4D
Legal plays: 4D
>Player1 plays 4D.
Player2 plays AH.
Player3 discards KS.
Player4 plays JS.
Cards on the table:
Clubs: 3 4 5 6 7 8 9 T J Q K
Diamonds: 4 5 6 7 8 9 T J Q K
Hearts: A 2 3 4 5 6 7 8 9 T J Q K
Spades: 2 3 4 5 6 7 8 9 T J
Your hand: QS
Legal plays: QS
>Player1 plays QS.
Player2 plays 3D.
Player1's discards: AS AD 2D AC
Player1's score: 5 + 5 = 10
Player2's discards:
Player2's score: 16 + 0 = 16
Player3's discards: KS
Player3's score: 49 + 13 = 62
Player4's discards: 2C
Player4's score: 32 + 2 = 34
A new round begins. It's Player4's turn to play.
Player4 plays 7S.
Cards on the table:
Clubs:
Diamonds:
Hearts:
Spades: 7
Your hand: JD 7C 5D JH KH 8D 3H 5S 5C JS 9H TD 3D
Legal plays: 7C
>Player1 plays 7C.
Player2 plays 8C.
Player3 plays 7H.
Player4 plays 6C.
Cards on the table:
Clubs: 6 7 8
Diamonds:
Hearts: 7
Spades: 7
Your hand: JD 5D JH KH 8D 3H 5S 5C JS 9H TD 3D
Legal plays: 5C
>Player1 plays 5C.
Player2 plays 4C.
Player3 plays 6S.
Player4 plays 8H.
Cards on the table:
Clubs: 4 5 6 7 8
Diamonds:
Hearts: 7 8
Spades: 6 7
Your hand: JD 5D JH KH 8D 3H 5S JS 9H TD 3D
Legal plays: 5S 9H
>Player1 plays 9H.
Player2 plays 6H.
Player3 plays 5H.
Player4 plays TH.
Cards on the table:
Clubs: 4 5 6 7 8
Diamonds:
Hearts: 5 6 7 8 9 T
Spades: 6 7
Your hand: JD 5D JH KH 8D 3H 5S JS TD 3D
Legal plays: JH 5S
>Player1 plays JH.
Player2 plays QH.
Player3 plays 4H.
Player4 plays 7D.
Cards on the table:
Clubs: 4 5 6 7 8
Diamonds: 7
Hearts: 4 5 6 7 8 9 T J Q
Spades: 6 7
Your hand: JD 5D KH 8D 3H 5S JS TD 3D
Legal plays: KH 8D 3H 5S
>Player1 plays KH.
Player2 discards KC.
Player3 plays 9C.
Player4 plays 6D.
Cards on the table:
Clubs: 4 5 6 7 8 9
Diamonds: 6 7
Hearts: 4 5 6 7 8 9 T J Q K
Spades: 6 7
Your hand: JD 5D 8D 3H 5S JS TD 3D
Legal plays: 5D 8D 3H 5S
>Player1 plays 8D.
Player2 discards JC.
Player3 plays TC.
Player4 plays 9D.
Cards on the table:
Clubs: 4 5 6 7 8 9 T
Diamonds: 6 7 8 9
Hearts: 4 5 6 7 8 9 T J Q K
Spades: 6 7
Your hand: JD 5D 3H 5S JS TD 3D
Legal plays: 5D 3H 5S TD
>Player1 plays TD.
Player2 discards AS.
Player3 plays 8S.
Player4 plays 9S.
Cards on the table:
Clubs: 4 5 6 7 8 9 T
Diamonds: 6 7 8 9 T
Hearts: 4 5 6 7 8 9 T J Q K
Spades: 6 7 8 9
Your hand: JD 5D 3H 5S JS 3D
Legal plays: JD 5D 3H 5S
>Player1 plays JD.
Player2 plays TS.
Player3 plays 3C.
Player4 discards 2H.
Cards on the table:
Clubs: 3 4 5 6 7 8 9 T
Diamonds: 6 7 8 9 T J
Hearts: 4 5 6 7 8 9 T J Q K
Spades: 6 7 8 9 T
Your hand: 5D 3H 5S JS 3D
Legal plays: 5D 3H 5S JS
>Player1 plays JS.
Player2 plays QD.
Player3 plays 2C.
Player4 plays QS.
Cards on the table:
Clubs: 2 3 4 5 6 7 8 9 T
Diamonds: 6 7 8 9 T J Q
Hearts: 4 5 6 7 8 9 T J Q K
Spades: 6 7 8 9 T J Q
Your hand: 5D 3H 5S 3D
Legal plays: 5D 3H 5S
>Player1 plays 5S.
Player2 plays KS.
Player3 plays 4S.
Player4 plays 3S.
Cards on the table:
Clubs: 2 3 4 5 6 7 8 9 T
Diamonds: 6 7 8 9 T J Q
Hearts: 4 5 6 7 8 9 T J Q K
Spades: 3 4 5 6 7 8 9 T J Q K
Your hand: 5D 3H 3D
Legal plays: 5D 3H
>Player1 plays 5D.
Player2 plays KD.
Player3 plays 2S.
Player4 discards AD.
Cards on the table:
Clubs: 2 3 4 5 6 7 8 9 T
Diamonds: 5 6 7 8 9 T J Q K
Hearts: 4 5 6 7 8 9 T J Q K
Spades: 2 3 4 5 6 7 8 9 T J Q K
Your hand: 3H 3D
Legal plays: 3H
>Player1 plays 3H.
Player2 plays AC.
Player3 plays 4D.
Player4 discards AH.
Cards on the table:
Clubs: A 2 3 4 5 6 7 8 9 T
Diamonds: 4 5 6 7 8 9 T J Q K
Hearts: 3 4 5 6 7 8 9 T J Q K
Spades: 2 3 4 5 6 7 8 9 T J Q K
Your hand: 3D
Legal plays: 3D
>Player1 plays 3D.
Player2 plays 2D.
Player3 discards QC.
Player1's discards:
Player1's score: 10 + 0 = 10
Player2's discards: KC JC AS
Player2's score: 16 + 25 = 41
Player3's discards: QC
Player3's score: 62 + 12 = 74
Player4's discards: 2H AD AH
Player4's score: 34 + 4 = 38
A new round begins. It's Player3's turn to play.
Player3 plays 7S.
Player4 plays 6S.
Cards on the table:
Clubs:
Diamonds:
Hearts:
Spades: 6 7
Your hand: 8S TH TS 2C AD 7H 2H QH QS 8H AH 7D 3D
Legal plays: 8S 7H 7D
>Player1 plays 8S.
Player2 plays 9S.
Player3 plays 5S.
Player4 plays 7C.
Cards on the table:
Clubs: 7
Diamonds:
Hearts:
Spades: 5 6 7 8 9
Your hand: TH TS 2C AD 7H 2H QH QS 8H AH 7D 3D
Legal plays: TS 7H 7D
>Player1 plays TS.
Player2 plays JS.
Player3 plays 4S.
Player4 plays 6C.
Cards on the table:
Clubs: 6 7
Diamonds:
Hearts:
Spades: 4 5 6 7 8 9 T J
Your hand: TH 2C AD 7H 2H QH QS 8H AH 7D 3D
Legal plays: 7H QS 7D
>Player1 plays QS.
Player2 plays 3S.
Player3 discards KC.
Player4 plays 5C.
Cards on the table:
Clubs: 5 6 7
Diamonds:
Hearts:
Spades: 3 4 5 6 7 8 9 T J Q
Your hand: TH 2C AD 7H 2H QH 8H AH 7D 3D
Legal plays: 7H 7D
>Player1 plays 7D.
Player2 plays KS.
Player3 discards TD.
Player4 plays 8D.
Cards on the table:
Clubs: 5 6 7
Diamonds: 7 8
Hearts:
Spades: 3 4 5 6 7 8 9 T J Q K
Your hand: TH 2C AD 7H 2H QH 8H AH 3D
Legal plays: 7H
>Player1 plays 7H.
Player2 plays 6D.
Player3 plays 6H.
Player4 plays 9D.
Cards on the table:
Clubs: 5 6 7
Diamonds: 6 7 8 9
Hearts: 6 7
Spades: 3 4 5 6 7 8 9 T J Q K
Your hand: TH 2C AD 2H QH 8H AH 3D
Legal plays: 8H
>Player1 plays 8H.
Player2 plays 9H.
Player3 discards JH.
Player4 plays 2S.
Cards on the table:
Clubs: 5 6 7
Diamonds: 6 7 8 9
Hearts: 6 7 8 9
Spades: 2 3 4 5 6 7 8 9 T J Q K
Your hand: TH 2C AD 2H QH AH 3D
Legal plays: TH
>Player1 plays TH.
Player2 plays 5D.
Player3 plays AS.
Player4 plays 8C.
Cards on the table:
Clubs: 5 6 7 8
Diamonds: 5 6 7 8 9
Hearts: 6 7 8 9 T
Spades: A 2 3 4 5 6 7 8 9 T J Q K
Your hand: 2C AD 2H QH AH 3D
Legal plays:
>Player1 discards AD.
Player2 plays 5H.
Player3 plays 4H.
Player4 plays 4D.
Cards on the table:
Clubs: 5 6 7 8
Diamonds: 4 5 6 7 8 9
Hearts: 4 5 6 7 8 9 T
Spades: A 2 3 4 5 6 7 8 9 T J Q K
Your hand: 2C 2H QH AH 3D
Legal plays: 3D
>Player1 plays 3D.
Player2 plays 4C.
Player3 plays 3H.
Player4 plays 3C.
Cards on the table:
Clubs: 3 4 5 6 7 8
Diamonds: 3 4 5 6 7 8 9
Hearts: 3 4 5 6 7 8 9 T
Spades: A 2 3 4 5 6 7 8 9 T J Q K
Your hand: 2C 2H QH AH
Legal plays: 2C 2H
>Player1 plays 2H.
Player2 discards JD.
Player3 plays 9C.
Player4 discards JC.
Cards on the table:
Clubs: 3 4 5 6 7 8 9
Diamonds: 3 4 5 6 7 8 9
Hearts: 2 3 4 5 6 7 8 9 T
Spades: A 2 3 4 5 6 7 8 9 T J Q K
Your hand: 2C QH AH
Legal plays: 2C AH
>Player1 plays 2C.
Player2 plays TC.
Player3 plays 2D.
Player4 plays AC.
Cards on the table:
Clubs: A 2 3 4 5 6 7 8 9 T
Diamonds: 2 3 4 5 6 7 8 9
Hearts: 2 3 4 5 6 7 8 9 T
Spades: A 2 3 4 5 6 7 8 9 T J Q K
Your hand: QH AH
Legal plays: AH
>Player1 plays AH.
Player2 discards KD.
Player3 discards QC.
Player4 discards KH.
Cards on the table:
Clubs: A 2 3 4 5 6 7 8 9 T
Diamonds: 2 3 4 5 6 7 8 9
Hearts: A 2 3 4 5 6 7 8 9 T
Spades: A 2 3 4 5 6 7 8 9 T J Q K
Your hand: QH
Legal plays:
>Player1 discards QH.
Player2 discards QD.
Player1's discards: AD QH
Player1's score: 10 + 13 = 23
Player2's discards: JD KD QD
Player2's score: 41 + 36 = 77
Player3's discards: KC TD JH QC
Player3's score: 74 + 46 = 120
Player4's discards: JC KH
Player4's score: 38 + 24 = 62
Player1 wins!
